app/components/ui/molecules/StaffGrid.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import styled from 'styled-components'
+import StaffCard from './StaffCard'
+
+const { Flex, Box } = require('grid-styled')
+
+const Grid = styled(Flex)`
+  flex-flow: row wrap;
+  justify-content: space-between;
+`
+const Card = styled(Box)`
+  padding-top: 20px;
+`
+
+const StaffGrid = ({ staff }) => (
+  <Grid>
+    {staff.map(person => (
+      <Card px={2} width={[1 / 2, 1 / 2, 1 / 3]}>
```

Comment:
I had a comment here but must not have added it to the review properly.

Perhaps we could do `px={3}` so that the horizontal spacing in between cards (i.e. gutters between columns) is `24px` & therefore the same as the vertical spacing between cards (i.e. the gutters in between rows)